src/main/scala/com/typesafe/scalalogging/Logger.scala

```diff
@@ -112,6 +112,8 @@ final class Logger private (val underlying: Underlying) extends Serializable {
 
   def error(marker: Marker, message: String, args: Any*): Unit = macro LoggerMacro.errorMessageArgsMarker
 
+  def whenError(body: Unit): Unit = macro LoggerMacro.errorCode
```

Comment:
would you be able to add this to the README?

Reply:
I added an example for this.